talentlink-connector/config/webservice-clients.yaml

```diff
@@ -0,0 +1,29 @@
+WebServiceClients:
+  documents:
+    Id: 1845780D24F6B50E
+    ServiceClass: com.mrted.ws.client.DocumentWebService_Service
+    WsdlUrl: https://api3.lumesse-talenthub.com/HRIS/SOAP/Document?WSDL
+    WsCallLibrary: CXF
+    Features:
+    - ch.ivyteam.ivy.webservice.exec.cxf.feature.WSSecurityAuthenticationFeature
+    - com.axonivy.connector.talentlink.features.ApiFeature
+    Properties:
+      username: ${ivy.var.talentlink-connector.username}
+      password: ${decrypt:\u00AC\u0078\u004A\u00A5\u000B\u0039\u0050\u0072\u0026\u0091\u000E\u0012\u00D2\u0048\u00F6\u009F\u0012\u00CB\u0048\u00B2\u00B2\u0098\u001A\u00BE\u0098\u0070\u0063\u0067\u0010\u00A4\u00A9\u00F5\u002E\u0097\u0021\u0041\u00D7\u001E\u00A5\u0049\u003B\u0065\u0001\u006C\u00EF\u0024\u00D4\u0008\u0031\u0036\u0081\u00DD\u0096\u0013\u00CE\u0016\u00B5\u00FD\u0049\u00D0\u00BE\u004F\u0031\u0032\u00E7\u00B2\u0014\u000D\u00E1\u00DA\u00AB\u00DE\u006D\u00CD\u0045\u0065\u005C\u0047\u0074\u00D6\u0092\u0088\u001D\u00FE\u0057\u00E9\u003F\u0017\u007A\u006D\u0028\u0077\u00FE\u004D\u00D7\u002C\u0078\u002E\u0003\u0066\u00B7\u0060\u00A5\u0090\u00A1\u00FF\u00DE\u0009\u0005\u0061\u0013\u0084\u0045\u00D3\u00D1\u00B0\u0044\u006F\u0095\u0082\u00DF\u0007\u0089\u004C\u002D\u0004\u007C\u003E\u007E\u00F2\u0028\u001B\u007B\u006F\u00B5\u0062\u0083\u0020\u0097\u0037\u00A1\u0037\u008E\u00B0\u006E\u0016\u00F7\u0052\u0042\u00DE\u00E8\u00A7\u000E\u0046\u00B9\u00BD\u0066\u001F\u0069\u0003\u0006\u00DB\u00BA\u00D3\u00C2\u004E\u0047\u000E\u0013\u00A2\u003B\u00B0\u0053\u000F\u00DB\u00C2\u00C1\u003E\u007F\u005E\u0083\u0052\u0072\u00A1\u00ED\u005A\u0013\u00B0\u002C\u0068\u00CB\u006E\u0083\u0060\u0020\u00CE\u00DB\u0037\u009E\u00CB\u00E1\u00F5\u00D7\u00F9\u00F5\u00A2\u0006\u0010\u000A\u00A8\u00E0\u00F6\u00FC\u0079\u0061\u00BC\u006C\u005F\u00CB\u00A3\u00DA\u00F1\u002D\u0002\u0022\u00B1\u0011\u00D2\u006F\u001B\u0065\u00BB\u00C3\u009C\u00F4\u00DA\u006A\u00A6\u0093\u00D1\u00F4\u0006\u001F\u00E5\u0084\u00A7\u00CB\u0092\u00E6\u0058\u00E2\u00D9\u0091\u00CC\u00B5\u00DE}
+    Endpoints:
+      DocumentWebServicePort:
+      - https://api3.lumesse-talenthub.com/HRIS/SOAP/Document
+  queues:
```

Comment:
can you share the idea behind having REST + SOAP service capabilities demonstrated in here?
- especially the 'queue' service seems to be demonstrated with REST and SOAP

personally I'd think that if we have REST interface, it's preferable to use only this rather than the SOAP based approach which is older and quickly running out of fashion. Or does the SOAP service have any advantage we don't have when using RESt? :thinking:

Reply:
for me it's ok to keep them:
- just note that the connector gets larger by this ... the git repo is already kind of slow here ... as we seem to have these binaries already since quite a while in the history.
- reading the official docs, it looks as if SOAP is only in maintenance mode. So it will quickly die.  ... so if you have no real usage for it in your project, I'd consider to remove it completely. 
- another idea: maybe we should include in a first version for documentation reasons... old api users can still install this first state; but we remove it rather soon from the product in sub-sequent relases, so we don't have to maintain it